pgbulk/core.py

```diff
@@ -63,7 +64,6 @@ def _psycopg_version() -> Tuple[int, int, int]:
 
 
 if psycopg_maj_version == 2:
-    from psycopg2.extensions import AsIs as Literal  # type: ignore
```

Comment:
Unused, but it seems Ruff/Pyright thought it was due to the above `typing.Literal`